mlir/lib/Dialect/Rock/Transforms/GridwiseGemmToBlockwise.cpp

```diff
@@ -2551,6 +2551,117 @@ struct GridwiseGemmAccelRewritePattern
     : public OpRewritePattern<GridwiseGemmAccelOp> {
   using OpRewritePattern<GridwiseGemmAccelOp>::OpRewritePattern;
 
+  // Generate only the compute loop, i.e., we assume here that all
+  // the data that we need is already in LDS
+  void generateComputeLoop(
+      Location loc, PatternRewriter &b,
+      const std::unique_ptr<rock::accel::AccelEmitter> &accelEmitterPtr,
+      Value regsA, Value regsB, Value regsC, StringAttr arch,
+      GemmFeaturesAttr features,
+      const RockAccelTuningParamAttrInterface tuningParams) const {
```

Comment:
Is there a reason for passing `tuningParams` by value and not by reference?

Reply:
Changed that to const ref. THanks